photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/GameTrainingDriver.scala

```diff
@@ -363,6 +363,12 @@ object GameTrainingDriver extends GameDriver {
       readValidationData(avroDataReader, featureIndexMapLoaders)
     }
 
+    val interceptIndices = featureIndexMapLoaders.map {
+      case (featureShardId, indexMapLoader) =>
+        indexMapLoader.indexMapForDriver().getIndex(Constants.INTERCEPT_KEY)
+      case _ => -1
+    }.toSeq
```

Comment:
See above comments - we should either have a `Map[CoordinateId, Option[Int]]` or encode the information elsewhere in a parameter that we're already passing to the `GameEstimator`.

I'm not sure where we could encode this information in existing parameters - let me think about it. While I do, you should feel free to ignore that advice and focus on the `Map[CoordinateId, Option[Int]]`.